src/main/webapp/stylesheets/teammatesCommon.css

```diff
@@ -862,9 +862,13 @@ Prevents content overflow in navbar. */
     padding: 20px 40px;
 }
 
-#student-data-spreadsheet .panel-body {
-    padding: 5px;
-    margin: 10px;
+.student-existing-data-spreadsheet .panel-default {
+    margin-left: 15px;
+    margin-right: 15px;
+}
+
+.student-enroll-spreadsheet {
+    margin: 15px;
```

Comment:
Why the different rules which increase the number of class names?

Reply:
The first line already has a `15px` margin as shown here:
![image](https://user-images.githubusercontent.com/25320782/41391901-eb53476e-6fcf-11e8-9b7b-79a1a781a5b5.png)

The `student-enroll-spreadsheet` also has a `15px` margin:
![image](https://user-images.githubusercontent.com/25320782/41391918-0a348efe-6fd0-11e8-9b92-699ad303e79c.png)

If I apply the same styles for two spreadsheets, then it will have an inconsistent `30px` gap between the first line and the next spreadsheet interface. That is why I separated them and only applied `15px` to both `margin-left` and `margin-right` to the `student-existing-data-spreadsheet`.

Reply:
Then how about removing the top margin for both and keep the bottom margin? That would achieve the same result.